### ISSUE TITLE
,WCAG 1.3.1,WCAG 4.1.2,Best Practice: ARIA role should be appropriate for the element (.sidenav__list--student[aria-label="student"][role="navigation"])

### ISSUE THREAD

**Issue by zcjess** [zcjess/GradeCraft#21]
#### Issue
`ARIA role should be appropriate for the element` ([aria-allowed-role](https://dequeuniversity.com/rules/axe/4.6/aria-allowed-role?application=msftAI))
#### Target application
[Michael Skib - testing](https://staging.gradecraft.com/courses/1144/assignments)
#### Element path
.sidebar-container > .sidenav__list--student[aria-label="student"][role="navigation"]
#### Snippet
`<ul aria-label="student" role="navigation" class="sidenav__list--student">`
#### Environment
Chrome version 115.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
